os/net/lwip/src/core/tcp_in.c

```diff
@@ -596,8 +607,10 @@ static err_t tcp_process(struct tcp_pcb *pcb)
 			pcb->flags &= ~TF_ACK_DELAY;
 			return ERR_RST;
 		} else {
-			LWIP_DEBUGF(TCP_INPUT_DEBUG, ("tcp_process: unacceptable reset seqno %" U32_F " rcv_nxt %" U32_F "\n", seqno, pcb->rcv_nxt));
-			LWIP_DEBUGF(TCP_DEBUG, ("tcp_process: unacceptable reset seqno %" U32_F " rcv_nxt %" U32_F "\n", seqno, pcb->rcv_nxt));
+			LWIP_DEBUGF(TCP_INPUT_DEBUG, ("tcp_process: unacceptable reset seqno %"U32_F" rcv_nxt %"U32_F"\n",
```

Comment:
what is your modification? There is nothing new.

Reply:
tcp_process() checks sequence number and current state when it receives the packet. if an incoming seqence number is between recv next and recv next + recv wnd then it sends ack again instead of accepting TCP_RST

Reply:
@davidfather U32_F is interpreted as "u" so basically "U32_F" and " U32_F " are same. but I am sorry to make you confused.